thin-replica-server/include/thin-replica-server/thin_replica_impl.hpp

```diff
@@ -39,10 +39,12 @@
 #include "thin_replica.grpc.pb.h"
 #include "subscription_buffer.hpp"
 #include "trs_metrics.hpp"
+#include <experimental/filesystem>
```

Comment:
Why do you need to use "experimental/filesystem"? ecursive_directory_iterator exists in "filesystem", too.